### PR TITLE
Add next bundle analyzer to store

### PR DESCRIPTION
**Pull request by gustaveen**
## Describe your changes
Add next bundle analyzer to store

### nodejs
Screenshot 2023-02-02 at 15 30 53

### client
Screenshot 2023-02-02 at 15 31 18

### edge
Screenshot 2023-02-02 at 15 30 46

## Justify why they are needed
Get some stats on our bundle size

## Checklist before requesting a review

- [ ] I have performed a self-review of my code

### CODE REVIEW

**Comment by gustaveen** [comment]
Current dependencies on/for this PR:
* main
  * **PR #1514** <a href="https://app.graphite.dev/github/pr/HedvigInsurance/racoon/1514" target="_blank">Graphite</a>  👈

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/HedvigInsurance/racoon/1514?utm_source=stack-comment).

File: apps/store/README.md [hunk 1]
```diff
@@ -77,3 +77,11 @@ The `next` query parameter is required. When creating a new shop session, we bas
...
 
 You enable a feature flag by supplying the `NEXT_PUBLIC_FEATURE_XXX` environment variable. For example, to enable the `INSURELY` feature, you would set the `NEXT_PUBLIC_FEATURE_INSURELY=true`.
+
+## Analyze bundle
+
+To analyze bundle size run the following command and this will open a report in the browser
+
+```bash
+yarn workspace store analyze-bundle
+```
```
Comment (gustaveen) [inline apps/store/README.md:87]: Wasn't sure in what Readme to put it in

**Review by robinandeer — CHANGES_REQUESTED** [review]
Great stuff! I bet the client bundle analysis will come in handy

Just noticed some added dependencies that must be a mistake

File: apps/store/package.json [hunk 2]
```diff
@@ -69,6 +71,7 @@
     "@testing-library/user-event": "14.4.3",
     "@types/node": "18.11.18",
     "@types/react": "18.0.27",
+    "add": "2.0.6",
```
[the remark this answers is not in the document]
Comment (gustaveen) [reply apps/store/package.json]: haha, I had apparently run `yarn workspace store add yarn add @next/bundle-analyzer` 🤣

**Review by robinandeer — APPROVED** [review]
[cool](https://app.graphite.dev/macros/HedvigInsurance)

**Review by alebedev — APPROVED** [review]
Looks good!  Great that we aren't using webpack-bundle-analyzer directly, this setup is so much simpler

**Pull request merged**